packages/victory-histogram/src/victory-histogram.js

```diff
@@ -0,0 +1,115 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { Bar } from "victory-bar";
+import {
+  Helpers,
+  VictoryLabel,
+  VictoryContainer,
+  VictoryTheme,
+  CommonProps,
+  addEvents,
+  PropTypes as CustomPropTypes
+} from "victory-core";
+import { getBaseProps, getData, getDomain, getFormattedData } from "./helper-methods";
+
+const fallbackProps = {
+  width: 450,
+  height: 300,
+  padding: 50
+};
+
+const defaultData = [];
+
+export class VictoryHistogram extends React.Component {
+  static animationWhitelist = ["data", "domain", "height", "padding", "style", "width"];
+
+  static displayName = "VictoryHistogram";
+
+  static role = "histogram";
+
+  static defaultTransitions = {
+    onLoad: {
+      duration: 2000,
+      before: () => ({ _y: 0, _y1: 0, _y0: 0 }),
+      after: (datum) => ({ _y: datum._y, _y1: datum._y1, _y0: datum._y0 })
+    },
+    onExit: {
+      duration: 500,
+      before: () => ({ _y: 0, yOffset: 0 })
+    },
+    onEnter: {
+      duration: 500,
+      before: () => ({ y: 0 }),
```

Comment:
In `VictoryBar` the `onEnter` transition uses ` before: () => ({ _y: 0, _y1: 0, _y0: 0 })`. Maybe changing this would correct the entrance issue in histograms

Reply:
good catch, fixed this!